static/app/views/issueList/issueSearchWithSavedSearches.spec.tsx

```diff
@@ -23,6 +23,10 @@ describe('IssueSearchWithSavedSearches', () => {
       url: '/organizations/org-slug/searches/',
       body: [savedSearch],
     });
+    MockApiClient.addMockResponse({
```

Comment:
Ignore all of `MockApiClient.addMockResponse({...})` additions made in this PR - [this prevents CI from failing](https://develop.sentry.dev/development/testing/#api-fixtures)